rpcs3/Emu/RSX/VK/VulkanAPI.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#define VK_USE_PLATFORM_WIN32_KHR
```

Comment:
Seems like it'll take some more work to get it to build on *nix it seems. I forgot about dependancy on some windows stuff that i'm currently using (hInstance, hWnd).